src/session.ts

```diff
@@ -691,53 +629,15 @@ export class SessionManager implements Middleware {
             SessionStatus.Failed);
     }
 
-    private changePowerShellExePath(exePath: string) {
+    private changePowerShellDefaultVersion(exePath: IPowerShellExeDetails) {
```

Comment:
This looks like it would be a simple `async` method now

Reply:
fixed